Composer/packages/adaptive-flow/src/adaptive-flow-editor/AdaptiveFlowEditor.tsx

```diff
@@ -200,9 +192,9 @@ const VisualDesigner: React.FC<VisualDesignerProps> = ({ onFocus, onBlur, schema
                       }}
                     />
                   </div>
-                </FlowToolbar>
-              </MarqueeSelection>
-            </SelectionContext.Provider>
+                </MarqueeSelection>
```

Comment:
What is the reason for reordering the components?

Reply:
Hi @OEvgeny, we found duplicated detached elements in memory and narrowed down the problem to these two components. We couldn't find the exact reason, but the issue was fixed after reordering the components.